project-reports/2023/2023-Q2-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: 2023 Q2 Hyperledger Iroha
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Currently, active maintenance of Iroha 1 has stopped and there are no plans for major releases. However, discussions to release the current RC continue because it will require more than one team switching from everyday tasks to focus on this.
+
+All Iroha 2 developers are now fully onboarded and at maximum productivity.
+
+Since development of the Ursa project has been discontinued, the Iroha team is rewriting its essential parts to integrate in the Iroha codebase. In addition, thanks to Arun S M, the Iroha 2 BCE team will start collaborating with the Hyperledger Blockchain Explorer team.
+
+# Questions/Issues for the TSC
+
+None.
+
+# Releases
+
+Past quarter releases:
+
+* v2.0.0-pre-rc.13
+* v2.0.0-pre-rc.15
+
+# Overall Activity in the Past Quarter
+
+According to the [Iroha Activity Dashboard](https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Firoha/dashboard;subTab=technical?time=%7B%22from%22:%222023-01-01T07:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222023-03-31T07:00:00.000Z%22%7D) for the first quarter of 2023 (January-March), the Iroha repo has 129 PRs submitted by 28 contributors.
+
+## Iroha v1
+
+Documentation restoration is in progress.
+
+Grzegorz Bazior started an Iroha 1 mentorship: "[extend queries with optional arguments](https://mentorship.lfx.linuxfoundation.org/project/06a25b14-a56e-4294-a89d-05f1dc74106c)".
+
+## Iroha v2
+
```

Comment:
### Documentation updates

We have conducted a revision of the original Iroha v2 tutorial to identify issues and outdated content. We have onboarded two new technical writers and have constructed a roadmap of advanced explanations. Work is underway for addition of detailed `iroha_client_cli` documentation as well as additional elements regarding architectural design. We plan to expand the tutorial with a section that was used as a formal reports for one of our internal projects.